cartography/intel/okta/origins.py

```diff
@@ -1,126 +1,141 @@
+    transformed_origins = _transform_okta_origins(origins)
+    _load_okta_origins(neo4j_session, transformed_origins, common_job_parameters)
+    _cleanup_okta_origins(neo4j_session, common_job_parameters)
 
 
 @timeit
-def transform_trusted_origins(data: str) -> List[Dict]:
+async def _get_okta_origins(okta_client: OktaClient) -> List[OktaTrustedOrigin]:
     """
-    Transform trusted origin data returned by Okta Server
-    :param data: json response
-    :return: Array of dictionary containing trusted origins properties
+    Get Okta origins list from Okta
+    :param okta_client: An Okta client object
+    :return: List of Okta origins
     """
-    ret_list: List[Dict] = []
-
-    json_data = json.loads(data)
-    for origin_data in json_data:
-        props = {}
-        props["id"] = origin_data["id"]
-        props["name"] = origin_data["name"]
-        props["origin"] = origin_data["origin"]
+    output_origins = []
+    query_parameters = {"limit": 200}
+    origins, resp, _ = await okta_client.list_origins(query_parameters)
 
-        # https://developer.okta.com/docs/reference/api/trusted-origins/#scope-object
-        scope_types = []
-        for scope in origin_data.get("scopes", []):
-            scope_types.append(scope["type"])
+    output_origins += origins
+    while resp.has_next():
+        origins, _ = await resp.next()
+        output_origins += origins
+        logger.info(f"Fetched {len(origins)} origins")
+    return output_origins
 
-        props["scopes"] = scope_types
-        props["status"] = origin_data["status"]
-        props["created"] = origin_data.get("created", None)
-        props["created_by"] = origin_data.get("createdBy", None)
-        props["okta_last_updated"] = origin_data.get("lastUpdated", None)
-        props["okta_last_updated_by"] = origin_data.get("lastUpdatedBy", None)
 
-        ret_list.append(props)
-
-    return ret_list
+@timeit
+def _transform_okta_origins(
+    okta_origins: List[OktaTrustedOrigin],
+) -> List[Dict[str, Any]]:
+    """
+    Convert a list of Okta origins into a format for Neo4j
+    :param okta_origins: List of Okta origins
+    :return: List of origin dicts
+    """
+    transformed_origins: List[Dict] = []
+    logger.info(f"Transforming {len(okta_origins)} Okta origins")
+    for okta_origin in okta_origins:
+        origin_props = {}
+        origin_props["id"] = okta_origin.id
+        origin_props["created"] = okta_origin.created
+        origin_props["created_by"] = okta_origin.created_by
+        origin_props["last_updated"] = okta_origin.last_updated
+        origin_props["last_updated_by"] = okta_origin.last_updated_by
+        origin_props["name"] = okta_origin.name
+        origin_props["origin"] = okta_origin.origin
+        # Scopes have a specific type
+        # currently supported are CORS, Redirect, and IFrame
+        for scope in okta_origin.scopes:
+            apps = []
+            string_value = None
```

Comment:
`string_value` is never set. Can we delete this line and delete these lines too?

```python
origin_props["cors_value"] = string_value
...
origin_props["redirect_value"] = string_value
...
origin_props["iframe_value"] = string_value
```